taskcluster/tc-build-utils.sh

```diff
@@ -276,11 +276,30 @@ do_deepspeech_netframework_build()
     /p:Platform=x64 \
     /p:TargetFramework="uap10.0" \
     /p:OutputPath=bin/nuget/x64/uap10.0
+	
+  MSYS2_ARG_CONV_EXCL='/' "${MSBUILD}" \
+    DeepSpeechClient/DeepSpeechClient.csproj \
+    /p:Configuration=Release \
+    /p:Platform="Any CPU" \
```

Comment:
Do we really want `Any CPU` ? Is this build to provide cross-compilation ?

Reply:
FTR, we can't have Windows 32-bits support, because TensorFlow does not provide it.

Reply:
since it's can now run on raspbian using `netcore` - we need to use any cpu config. It also will not run on unsupported OS's like `win-x86`  because there is no `.so` for that runtime (when saying runtime I mean [this](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog) catalog), so not sure if we need to change something here